index.html

```diff
@@ -0,0 +1,244 @@
+                </div>
+                <p><a class=" blu">LET'S CONNECT</a></p>
+                <ul class="socials">
+                    <li class="link">
+                        <a href=""><img class="astaricks" src="images/Twitter.svg" alt="Twitter" /></a>
+                    </li>
+                    <li class="link">
+                        <a href=""><img class="astaricks" src="images/Linkedin.svg" alt="Linkedin" /></a>
+                    </li>
+                    <li class="link">
+                        <a href=""><img class="astaricks" src="images/Medium.svg" alt="Medium" /></a>
+                    </li>
+                    <li class="link">
+                        <a href=""><img class="astaricks" src="images/GitHub.svg" alt="Github" /></a>
+                    </li>
+                    <li class="link">
+                        <a href=""><img class="astaricks" src="images/Angel list.svg" alt="ANGEL LIST"></a>
+                    </li>
+                </ul>
+                <div class="initiate"><button type="key" class="tapbutton">Check my resume</button></div>
+            </div>
+            <div class="purple_square">
+                <div class="sidebyside">
+                    <li class="portrait">
+                        <h3 class="boldtext">Languages
+                            <div><img class="arrowkey" src="images/Icon - Chevron - Right - Larger.png" /></div>
+                        </h3>
+
+                        <div></div>
+                        <div></div>
+                        <div></div>
+
+
+                    </li>
+
+                    <div id="wrapper">
+                        <div id="inner">
+                            <div class="portrait">
+                                <li class="obtainer"> <img src="images/javascriptLogo.png">
+                                    <p>Javascript</p>
+                                </li>
+                            </div>
+                            <div class="portrait">
+                                <li class="obtainer"> <img src="images/html logo.png" />
+                                    <p>HTML</p>
+                                </li>
+                            </div>
+                            <div class="portrait">
+                                <li class="obtainer"> <img src="images/css logo.png">
+                                    <p>CSS</p>
+                                </li>
+                            </div>
+
+                        </div>
+                    </div>
+
+
+                    <h3 class="boldtext">Frameworks
+                        <div> <img class="arrowkey" src="images/Icon - Chevron - down - Larger.png" alt="down" /></div>
+                    </h3>

```

Comment:
It is not recommended to nest inside a header tag a `div` and a `img` tags.

Maybe because of this reason the arrow-down is not shown yet.

- [ ] Please make sure to display the arrow-down icons (Also, please double check the path and the name if are correct too);

![image](https://user-images.githubusercontent.com/83596135/203446781-817b5aee-d663-43ec-b0e0-2716576ebc31.png)

Reply:
- [ ] The arrow images are still broken. The name of the icons includes white spaces. I recommend renaming the icon's name in the images folder and using a short and easy name without white spaces (ex: arrow.png). Try to assign the same name to the src attribute (ex: src="arrow.png"). Kindly ensure that the icons are not broken by running the live server.